### PR TITLE
Added more command aliases and shorthands

### PR DESCRIPTION
**Pull request by rladenson**

### CODE REVIEW

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 1]
```diff
@@ -262,21 +262,21 @@ private async Task HandleSystemCommand(Context ctx)
...
+            else if (ctx.Match("name", "rename", "changename", "setname", "rn"))
                 await ctx.Execute<SystemEdit>(SystemRename, m => m.Name(ctx));
-            else if (ctx.Match("tag"))
+            else if (ctx.Match("tag", "t"))
                 await ctx.Execute<SystemEdit>(SystemTag, m => m.Tag(ctx));
-            else if (ctx.Match("servertag"))
+            else if (ctx.Match("servertag", "st", "stag"))
                 await ctx.Execute<SystemEdit>(SystemServerTag, m => m.ServerTag(ctx));
-            else if (ctx.Match("description", "desc", "bio"))
+            else if (ctx.Match("description", "desc", "bio", "text", "info"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:271]: why `text`?

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 2]
```diff
@@ -307,6 +307,13 @@ private async Task HandleSystemCommand(Context ctx)
                 await PrintCommandList(ctx, "systems", SystemCommands);
             else if (ctx.Match("groups", "gs", "g"))
                 await ctx.Execute<Groups>(GroupList, g => g.ListSystemGroups(ctx, null));
+            else if (ctx.Match("group"))
+                if (ctx.Match("list", "l"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:311]: This just does nothing if someone types `pk;s group`, not great. why not just add `group` to the match above that?

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 3]
```diff
@@ -323,6 +330,9 @@ private async Task HandleSystemCommandTargeted(Context ctx)
             }
             else if (ctx.Match("list", "l", "members"))
                 await ctx.Execute<SystemList>(SystemList, m => m.MemberList(ctx, target));
+            else if(ctx.Match("member", "m"))
+                if(ctx.Match("list", "l"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:334]: Same as above. Also, can you add `add`/`remove` here as well?

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 4]
```diff
@@ -342,6 +352,9 @@ private async Task HandleSystemCommandTargeted(Context ctx)
                 await ctx.Execute<System>(SystemInfo, m => m.Query(ctx, target));
             else if (ctx.Match("groups", "gs"))
                 await ctx.Execute<Groups>(GroupList, g => g.ListSystemGroups(ctx, target));
+            else if(ctx.Match("group", "g"))
+                if(ctx.Match("list", "l"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:356]: Same as above.

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 5]
```diff
@@ -262,21 +262,21 @@ private async Task HandleSystemCommand(Context ctx)
             // First, we match own-system-only commands (ie. no target system parameter)
             else if (ctx.Match("new", "create", "make", "add", "register", "init", "n"))
                 await ctx.Execute<System>(SystemNew, m => m.New(ctx));
-            else if (ctx.Match("name", "rename", "changename"))
+            else if (ctx.Match("name", "rename", "changename", "setname", "rn"))
                 await ctx.Execute<SystemEdit>(SystemRename, m => m.Name(ctx));
-            else if (ctx.Match("tag"))
+            else if (ctx.Match("tag", "t"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:267]: I wouldn't shorten `tag` to `t`

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 6]
```diff
@@ -430,28 +443,34 @@ private async Task HandleGroupCommand(Context ctx)
...
                     await ctx.Execute<Groups>(GroupDesc, g => g.GroupDescription(ctx, target));
                 else if (ctx.Match("add", "a"))
                     await ctx.Execute<Groups>(GroupAdd,g => g.AddRemoveMembers(ctx, target, Groups.AddRemoveOperation.Add));
-                else if (ctx.Match("remove", "rem", "r"))
-                    await ctx.Execute<Groups>(GroupRemove, g => g.AddRemoveMembers(ctx, target, Groups.AddRemoveOperation.Remove));
                 else if (ctx.Match("members", "list", "ms", "l"))
                     await ctx.Execute<Groups>(GroupMemberList, g => g.ListGroupMembers(ctx, target));
-                else if (ctx.Match("random"))
+                else if(ctx.Match("member", "m"))
+                    if(ctx.Match("list", "l"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:457]: Same as above.

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 7]
```diff
@@ -430,28 +443,34 @@ private async Task HandleGroupCommand(Context ctx)
...
-                if (ctx.Match("rename", "name", "changename", "setname"))
+                if (ctx.Match("rename", "name", "changename", "setname", "rn"))
                     await ctx.Execute<Groups>(GroupRename, g => g.RenameGroup(ctx, target));
                 else if (ctx.Match("nick", "dn", "displayname", "nickname"))
                     await ctx.Execute<Groups>(GroupDisplayName, g => g.GroupDisplayName(ctx, target));
                 else if (ctx.Match("description", "info", "bio", "text", "desc"))
                     await ctx.Execute<Groups>(GroupDesc, g => g.GroupDescription(ctx, target));
                 else if (ctx.Match("add", "a"))
                     await ctx.Execute<Groups>(GroupAdd,g => g.AddRemoveMembers(ctx, target, Groups.AddRemoveOperation.Add));
-                else if (ctx.Match("remove", "rem", "r"))
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:441]: Why was this removed?

File: PluralKit.Bot/Commands/CommandTree.cs [hunk 8]
```diff
@@ -430,28 +443,34 @@ private async Task HandleGroupCommand(Context ctx)
...
                 else if (ctx.Match("private", "priv"))
                     await ctx.Execute<Groups>(GroupPrivacy, g => g.GroupPrivacy(ctx, target, PrivacyLevel.Private));
-                else if (ctx.Match("delete", "remove", "destroy", "erase", "yeet"))
+                else if(ctx.Match("delete", "destroy", "erase", "yeet"))
                     await ctx.Execute<Groups>(GroupDelete, g => g.DeleteGroup(ctx, target));
+                else if (ctx.Match("remove", "rem", "rm"))
+                    if(!ctx.HasNext())
+                        await ctx.Execute<Groups>(GroupDelete, g => g.DeleteGroup(ctx, target));
+                    else
+                        await ctx.Execute<Groups>(GroupRemove, g => g.AddRemoveMembers(ctx, target, Groups.AddRemoveOperation.Remove));
```
Comment (spiralw) [inline PluralKit.Bot/Commands/CommandTree.cs:473]: Why?

**Comment by spiralw** [comment]
Can you fix the build issues?

**Review by spiralw — CHANGES_REQUESTED** [review]

**Comment by rladenson** [comment]
I *think* that's everything resolved?

**Comment by spiralw** [comment]
> I noticed remove was trying to be a valid option for delete group and delete members from group so I tried to make either be an option

i'd rather it just remove members, not delete the whole group, since it's never done that

> Um. That's inside HandleSystemCommandTargeted? Why would add/remove be in there?

oh. i was on mobile and misread the code, sorry

> it reads like [...] they're trying to pull up a specific group [in a specific system].

ah, I see. that should be a thing. (not in this PR, though)

I'll merge in a bit.
